betty/error.py

```diff
@@ -9,14 +9,20 @@
 BaseExceptionT = TypeVar('BaseExceptionT', bound=BaseException)
 
 
+def format_traceback(error: BaseException) -> str:
```

Comment:
Use the traceback module for this. I don't think there is a need for Betty to provide this at all.